django/db/models/fields/__init__.py

```diff
@@ -298,8 +312,8 @@ def _check_choices(self):
 
         return [
             checks.Error(
-                "'choices' must be an iterable containing "
-                "(actual value, human readable name) tuples.",
+                "'choices' must be an iterable of tuples or a dictionary "
+                "containing (actual value, human readable name) tuples.",
```

Comment:
These need updating in `ref/check.txt`

Reply:
Done, I didn't realize that existed. I wonder if we can automatically populate that files contents somehow?